From 5494bb6328dc4508c1e3f6a03ff945a713058484 Mon Sep 17 00:00:00 2001
From: Ivan Vecera <ivecera@redhat.com>
Date: Mon, 8 Sep 2014 16:14:32 -0400
Subject: [ethernet] be2net: add a description for counter rx_input_fifo_overflow_drop

Message-id: <1410192881-30702-70-git-send-email-ivecera@redhat.com>
Patchwork-id: 92472
O-Subject: [RHEL7.1 PATCH 69/78] be2net: add a description for counter rx_input_fifo_overflow_drop
Bugzilla: 1088575
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Nikolay Aleksandrov <nikolay@redhat.com>

BZ: #1088575

Upstream commit(s):
commit acbd6ff833da9d84e6596bfc94c5605a9c5b66da
Author: Sathya Perla <sathya.perla@emulex.com>
Date:   Tue Sep 2 09:56:48 2014 +0530

    be2net: add a description for counter rx_input_fifo_overflow_drop

    Signed-off-by: Sathya Perla <sathya.perla@emulex.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>

Signed-off-by: Ivan Vecera <ivecera@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/emulex/benet/be_ethtool.c b/drivers/net/ethernet/emulex/benet/be_ethtool.c
index c40f35c..d23b180 100644
--- a/drivers/net/ethernet/emulex/benet/be_ethtool.c
+++ b/drivers/net/ethernet/emulex/benet/be_ethtool.c
@@ -78,6 +78,11 @@ static const struct be_ethtool_stat et_stats[] = {
   * fifo must never overflow.
   */
  {DRVSTAT_INFO(rxpp_fifo_overflow_drop)},
+ /* Received packets dropped when the RX block runs out of space in
+  * one of its input FIFOs. This could happen due a long burst of
+  * minimum-sized (64b) frames in the receive path.
+  * This counter may also be erroneously incremented rarely.
+  */
  {DRVSTAT_INFO(rx_input_fifo_overflow_drop)},
  {DRVSTAT_INFO(rx_ip_checksum_errs)},
  {DRVSTAT_INFO(rx_tcp_checksum_errs)},
-- 
1.7.1